static/src/components/builder/components/DropdownInput/DropdownForm.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div class="row">
+    <h4>Dropdown Settings</h4>
+    <div class="form-group">
+      <input
+        id="add-label"
+        v-model="labelAdded"
+        type="checkbox"
+      >
+      <label for="add-label">
+        Add Dropdown Label
+      </label>
+      <input
+        v-if="labelAdded === true"
+        v-model="label"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </div>
+    <label
+      class="block-label"
+    >
+      Answer field name | <span class="label-tip">The field where the worker's answer is stored. Can be JSON path like a.b.c.</span>
+      <input
+        id="pyb-answer"
+        v-model="pybAnswer"
+        class="form-control form-control-sm"
+        type="text"
+      >
+    </label>
+    <label
+      class="block-label"
+    >
+      Initial Value | <span class="label-tip">Must be one of the configured choice values.</span>
```

Comment:
I think instead of showing an empty dropdown, Maybe Initial value input could be after all choices are selected.

Reply:
If we move it we can even remove the <span class="label-tip">Must be one of the configured choice values.</span> since its going to be clear to the user.